.eslintrc

```diff
@@ -37,5 +38,30 @@
         "space-before-function-paren": [1, "never"],
         "spaced-comment": [1, "always"],
         "react/prop-types": 1,
+        "react/display-name": [0, { "acceptTranspilerName": true }],
```

Comment:
I don't think all these extra rules are really needed, this is an example - not a boilerplate.